lib/jxl/enc_ac_strategy.cc

```diff
@@ -929,6 +930,10 @@ void ProcessRectACSNew(PassesEncoderState* JXL_RESTRICT enc_state,
   // when DCT8X8 is specified in the tree search.
   // 8x8 transforms have 10 variants, but every larger transform is just a DCT.
   float entropy_estimate[64] = {};
+  static const float k8x8mul1 = -0.38173536034815592f;
+  static const float k8x8mul2 = 1.0305692427138704f;
+  static const float k8x8base = 1.5349788369698298f;
+  const float mul8x8 = k8x8mul2 + k8x8mul1 / (butteraugli_target + k8x8base);
```

Comment:
A comment would be appreciated here. This formula seems to imply that for a higher target distance, the small 8x8 transforms get a 'penalty' in the form of overestimating their cost (while at low distances, they get a 'bonus' as their true cost gets underestimated), is that the idea here?

Reply:
added